### PR TITLE
docs(common): apply review comments

### PR DESCRIPTION
**Pull request by adamwardecki**

### CODE REVIEW

File: docs/docs/common/limits.md [hunk 1]
```diff
@@ -4,24 +4,23 @@ The limits we apply are based on our research on data operations and are set to
...
+| Field type | Limit | Notes |
+|--- |--- |--- |
+|text|32000 max character length||
+|string|128 max character length||
+|integer|32-bit (signed)||
+|float|64-bit|Double precision field with up to 15 decimal digit precision|
+|array|Can hold only Strings, Integer or Floats|Cannot contain Object type (JSON fields)|
+|relation|Can hold relations to max 1000 Data Objects from one selected Data Class|Relation can be linked to only one selected data class. You cannot have Relation field storing references to Data Objects from multiple different data classes.|
+|object|Any JSON object|Without limits, limited only by the max size of Data Object (32kB)|
+|file|binary data, max 128MB|This is a web server limit and can be worked around by sending larger files in chunks.|
```
Comment (23doors) [inline docs/docs/common/limits.md]: That's not exactly true. Max http post body limit is 128MB. So you can in theory send file1 of 128MB then in another request send second file2 of 128MB to same DO. But overall you cannot have larger files than 128MB in any way.

File: docs/docs/common/limits.md [hunk 2]
```diff
@@ -4,24 +4,23 @@ The limits we apply are based on our research on data operations and are set to
 
 ## Data Objects
 
-Limits for the the Data Object fields
+- Maximal size of a Data Object is `32kB`
+- Data Objects can have a max of `32` fields.
+- `16` of the fields can be indexed, `8` if both `order_index` and `filter_index` is used for each field; `text` and `file` fields can't be indexed).
```
Comment (23doors) [inline docs/docs/common/limits.md]: as noted in last PR, `object` cannot be indexed as well. 
Also, not all fields can have order index (array, geopoint, relation) as it doesn't make sense there so not sure if this is even worth mentioning.
In relatively recent additions, there is also `unique` that implies `filter_index` but it also cannot be applied to text, file, array, geopoint and relation.

File: docs/docs/common/limits.md [hunk 3]
```diff
@@ -4,24 +4,23 @@ The limits we apply are based on our research on data operations and are set to
...
 
-A limit for a file size is 128MB. This is a web server limit and can be worked around by sending larger files in chunks.
+| Field type | Limit | Notes |
+|--- |--- |--- |
+|text|32000 max character length||
+|string|128 max character length||
+|integer|32-bit (signed)||
+|float|64-bit|Double precision field with up to 15 decimal digit precision|
+|array|Can hold only Strings, Integer or Floats|Cannot contain Object type (JSON fields)|
+|relation|Can hold relations to max 1000 Data Objects from one selected Data Class|Relation can be linked to only one selected data class. You cannot have Relation field storing references to Data Objects from multiple different data classes.|
```
Comment (23doors) [inline docs/docs/common/limits.md]: But you can have multiple relations to circumvent that and it's not exactly a limitation, just a characteristic that relation and reference types are bound to specific class when declared.

**Comment by 23doors** [comment]
codecov doesn't seem to be correct ;)

File: docs/docs/common/limits.md (hunk 3 above)
Comment (adamwardecki) [reply docs/docs/common/limits.md]: Should I remove this entry altogether?

Comment (23doors) [reply docs/docs/common/limits.md]: Maybe just leave it at `Can hold ids of max 1000 Data Objects`?

File: docs/docs/common/limits.md (hunk 2 above)
Comment (adamwardecki) [reply docs/docs/common/limits.md]: Maybe I'll add this info to another section on filtering and ordering which we should port from the legacy docs?

Comment (23doors) [reply docs/docs/common/limits.md]: Makes sense!

**Comment by adamwardecki** [comment]
@maciejkorsan Can you review?

**Comment by maciejkorsan** [comment]
LGTM

**Pull request merged**